README.md

```
@@ -1 +1,26 @@
+- Logs errors for corrupted files
+
+## Requirements
+
+- Python 3.9+
+- pandas
+- openpyxl
+- xlrd
+
+## Usage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)